functions-and-scope.js

```diff
@@ -16,7 +17,16 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6
 
+let cumLaudeGrades = 0;
+
+for(let i = 0; i <= grades.length-1; i++) {
 
+    if( i >= 8 ) {
```

Comment:
Ik mis in de if statement de check op de index van een entry in de array! Hier had
`if (grades[i] >= 8)`
moeten staan! :)